documentation/developing/modding_examples/blindfold_assist.mdx

```diff
@@ -125,7 +125,7 @@ And that's it! We repeat this code for the right stick, and everything downstrea
 
 ### Where/how is the camera defined?
 
-On the OpenGOAL side, the camera is represented by a few different types, but perhaps the most important one is the [`camera-master` type](https://github.com/OpenGOAL-Unofficial-Mods/opengoal-mod-base/blob/a775793814ad424bf9ba919aa5f5c39834a3101f/data/goal_src/jak1/engine/camera/camera-h.gc#L445-L508), and the commonly-used global variable `*camera*` of that type.
```

Comment:
I had internationally used permalinks here so that when master gets updated the line numbers aren't suddenly incorrect. We can do that with jak-project master still, I can grab the equivalent permalink when I'm at PC in a couple hours

Reply:
Yeah makes sense Ill look to fix that later as well if I get time. Whoever gets to it first. The main issue is the old Mod base is depreciated and was made private to prevent people from going there on accident.

Reply:
I got time now, I'm on it 🖖

Reply:
https://github.com/Zedb0T/open-goal.github.io/pull/1